darwin/gal/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "gal",
+    platforms: [
+        .iOS("11.0"),
+        .macOS("11.0")
+    ],
+    products: [
+        .library(name: "gal", targets: ["gal"])
+    ],
+    dependencies: [],
+    targets: [
+        .target(
+            name: "gal",
+            dependencies: [],
+            resources: [
+                .process("PrivacyInfo.xcprivacy")
+            ]
+        )
+    ]
+)
```

Comment:
we need the new line here 
![9e7544efd52bc2acdc7c9b36ba1c04e1](https://github.com/user-attachments/assets/e7359edc-2c5c-4c72-a6e5-ea5949e7cf33)

Reply:
![image](https://github.com/user-attachments/assets/90165200-ce1a-4156-8f07-2275d2e1e5c7)

Reply:
Running `wc -l darwin/gal/Package.swift` outputs `25` when it's `26` lines in the code editor and actual file, it's likely a configuration issue in Git (related to line-endings) or the editor in my local machine. I added another line from my end so it appears only one empty line at the end from remote.